static/sass/components/_intro.scss

```diff
@@ -0,0 +1,64 @@
+// Intro Blocks
+// ============
+// Two column (on wide screens) intro and aside
+// Can be seen on case studies
+
+// wrapping container for intro
+.intro-block {
+  @include grid;
```

Comment:
Ok, I was just wondering if the duplicate display is intentional. Does `supports` add specificity, so that overrides the flex setting?